### PR TITLE
Sidebar Paragraph and Table Section has 1fr 1fr (50/50) width

### PR DESCRIPTION
**Pull request by Andreas-Kainz**
The Spacing and Indent section has the same width
The Table section has now the same width

image

Change-Id: I0d1680b8fd0e97e109d86203fba7fe2ca0c82f51

### CODE REVIEW

**Comment by Andreas-Kainz** [comment]
Togehter with #6390 the alignment of the elements is now perfect.

Tested with firefox / chrome / edge / falcon

File: browser/css/jssidebar.css [hunk 1]
```diff
@@ -241,17 +241,14 @@ div.sidebar.ui-grid .checkbutton.sidebar,
...
 	justify-content: start;
 	row-gap: 4px;
 }
 
-#ParaPropertyPanel.sidebar #indentfieldbox {
-	display: grid;
-	row-gap: 4px;
-}
-
 #ParaPropertyPanel .sidebar.ui-grid-cell {
```
Comment (pedropintosilva) [inline browser/css/jssidebar.css:252]: Thanks! if you could have this part in a different commit or PR I could merge it right away

File: browser/css/jssidebar.css [hunk 2]
```diff
@@ -264,8 +261,14 @@ button#button2.ui-pushbutton.jsdialog.sidebar {
...
-	width: 158px;
+#TableEditPanel.sidebar.ui-grid #grid1,
+#ParaPropertyPanel.sidebar.ui-grid #grid1 {
+	grid-template-columns: 1fr 1fr !important;
+	column-gap: 16px;
+}
+
+#numberbullet {
+	width: 143px; /* to much conent break grid columns width */
 }
```
Comment (pedropintosilva) [inline browser/css/jssidebar.css:273]: I'm not sure about this yet. I wonder if changing "#ParaPropertyPanel.sidebar.ui-grid #box1.sidebar.ui-grid-cell" in the jssidebar.js  so justify-content is set to stretch would fix it . What do you think?

**Review by pedropintosilva — CHANGES_REQUESTED** [review]

**Comment by pedropintosilva** [comment]
Also some of these get fixed with https://github.com/CollaboraOnline/online/pull/6433

File: browser/css/jssidebar.css (hunk 1 above)
Comment (Andreas-Kainz) [reply browser/css/jssidebar.css:252]: see #6439

**Comment by Andreas-Kainz** [comment]
> Also some of these get fixed with #6433

Ok than I will check what #6433 changed and than offer a new PR.

File: browser/css/jssidebar.css (hunk 2 above)
Comment (Andreas-Kainz) [reply browser/css/jssidebar.css:273]: I played around, but the class I added it the simpliest method to align everything and make it look nicely cause each column use the same width. Therefore I made #6455

**Pull request closed without merging**